www/src/components/Footer.astro

```diff
@@ -0,0 +1,172 @@
+---
+import Stack from '@/components/Stack/Stack.astro';
+import Text from '@/components/Text/Text.astro';
+import { SITE_TITLE } from '@/consts';
+import { menu } from '@/site.config';
+import { Icon } from 'astro-icon/components';
+import { isLinkAbsolute } from '@/utils/string.utils';
+
+const year = new Date().getFullYear();
+const url = new URL(Astro.request.url);
+---
+
+<footer class="l-footer">
+  <Stack axis="y" gap={4}>
+    <Text variant="sm">
+      <span>© {year}</span>
+      <span>{SITE_TITLE}</span>
+    </Text>
+    <Text
+      variant="xs"
+      color="color-text-subdued"
+      as="a"
+      href="https://www.netlify.com"
+      target="_blank">Deploys by Netlify</Text
+    >
+  </Stack>
+
+  <nav class="l-footer__nav">
+    <div class="l-footer__subnav">
+      <Text variant="body" as="h2" color="color-text-subdued">Spec</Text>
+      <ul class="u-list-reset">
+        {
+          menu.about.map((link) => (
+            <li>
+              <a
+                aria-current={url.pathname === link.path ? 'page' : false}
+                data-astro-prefetch
+                class="link"
+                href={link.path}
+                aria-label={
+                  isLinkAbsolute(link.path)
+                    ? `Link opens in a new window (${link.path})`
```

Comment:
cc @kaelig I have to make an adjustment here because `aria-label` _replaces_ all meaningful text within, not merely appending. So as-written, screen readers would _only_ see “Link opens in a new window ([link path])” which isn’t a good accessible name (also they don’t need the address read out loud which could be noisy). `aria-description` is better for supplementary info that doesn’t replace the accessible name.

But for a simpler approach, I’m just giving the `<Icon />` inside a description that it’s an external link. That way the screen reader experience is 1:1 with the element structure